### PR TITLE
Adds "Jabroni Outfit" to ClothesMate Vendor.

### PR DESCRIPTION
**Pull request by TechnoAlchemisto**
:cl: Papa Bones
rscadd: A new, scandalous outfit is now available as contraband from the clothesmate vendor, check it out! Fulfill your deep, dark fantasies.
/:cl:

This has been rattling around my head ever since @Iamgoofball 's zombie gym tbh

FUCK
FUCK2
FUCK3

### CODE REVIEW

**Comment by optimumtact** [comment]
I'm an artist, A performance artist.

**Comment by Iamgoofball** [comment]
i'm the boss of this gym and I say no

**Comment by TechnoAlchemisto** [comment]
Fuck you.

**Comment by Iamgoofball** [comment]
Get out of that jabroni outfit, we'll settle this right here on the gym floor.

**Comment by WJohn** [comment]
:+1: This is super dumb.

**Comment by Bill-Luxe** [comment]
robust club 2 blocks down

File: code/modules/clothing/under/miscellaneous.dm [hunk 1]
```diff
@@ -79,6 +79,14 @@
 	item_color = "sexyclown"
 	can_adjust = 0
 
+/obj/item/clothing/under/jabroni
+	name = "Jabroni Outfit"
+	desc = "The leather club is two sectors down."
```
Comment (Iamgoofball) [inline code/modules/clothing/under/miscellaneous.dm]: FUCK YOU

**Comment by KorPhaeron** [comment]
happy birthday saeg

**Comment by Saegrimr** [comment]
This is the best late birthday present

**Comment by TechnoAlchemisto** [comment]
happy late birthday

**Pull request merged**